.github/workflows/test-kube.yml

```diff
@@ -0,0 +1,27 @@
+# This pipeline purpose is solely meant to run a subset of our test suites against a kube cluster
+name: kube
+
+on:
+  push:
+    branches:
+      - main
+      - 'release/**'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+env:
+  ROOTFUL: true
+
+jobs:
+  linux:
+    runs-on: "ubuntu-24.04"
+    timeout-minutes: 40
+    steps:
+      - uses: actions/checkout@v4.1.7
+        with:
+          fetch-depth: 1
+      - name: "Run Kube integration tests"
+        run: |
+          ./hack/build-integration-kube.sh
+          sudo ./_output/nerdctl exec nerdctl-test-control-plane bash -c -- 'export TMPDIR="$HOME"/tmp; mkdir -p "$TMPDIR"; cd /nerdctl-source; /usr/local/go/bin/go test ./cmd/nerdctl/ -test.kube'
```

Comment:
Maybe:
```suggestion
          sudo ./_output/nerdctl exec nerdctl-test-control-plane bash -c -- 'export TMPDIR="$HOME"/tmp; mkdir -p "$TMPDIR"; cd /nerdctl-source; /usr/local/go/bin/go test ./cmd/nerdctl/ -test.kube -test.run TestKube'
```

Reply:
Not sure I understand why?
`-test.kube` is already going to limit the run to whichever test is explicitly enabled for Kube, so, wouldn't that additional `-test.run TestKube` be moot in that context?

Am I missing something?

Reply:
> -test.kube is already going to limit the run to whichever test is explicitly enabled for Kube

This looks confusing and inconsistent with `-test.kill-daemon`

Reply:
Oh!
I see what you mean now.

I actually just mimicked the mechanism we have for `ipv6`: https://github.com/containerd/nerdctl/blob/main/Dockerfile#L357

https://github.com/containerd/nerdctl/blob/main/pkg/testutil/testutil.go#L548

Should I replace both flags?

- `-test.kube` => `-with-kube`
- `-test.ipv6` => `-with-ipv6`

Or something else?

Reply:
The `-test.ipv6` flag should be updated to follow the behavior of `-test.kill-daemon`?

If it is not possible the flag name should be like `-test.ipv6-only`?

Reply:
> The `-test.ipv6` flag should be updated to follow the behavior of `-test.kill-daemon`?

Yeah, I think what we want for both ipv6 and kube is to run these tests *only* - instead of *in addition* (like kill-daemon).


What about this for full clarity on these flags:

```
-test.enable-kill-daemon
-test.only-ipv6
-test.only-kube
```

?

Reply:
SGTM